go.mod

```diff
@@ -373,6 +373,7 @@ replace (
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.14
 	github.com/go-ozzo/ozzo-validation => github.com/go-ozzo/ozzo-validation v3.5.0+incompatible
 	github.com/go-stack/stack => github.com/go-stack/stack v1.8.0
+	github.com/go-task/slim-sprig => github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
```

Comment:
this looks like a new addition

Reply:
it seems is checked out in the vendor folder in this later commit https://github.com/kubernetes/kubernetes/pull/109111/commits/8f65cf28855e1b665e4242a09eb77bb84e8ebbef